python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -2,13 +2,24 @@
 import linecache
 import math
 import os
+import copy
 import json as json
 from lsst.sims.catalogs.measures.instance import register_class, register_method, compound
 from scipy.interpolate import InterpolatedUnivariateSpline
 from scipy.interpolate import UnivariateSpline
 from scipy.interpolate import interp1d
 
-__all__ = ["Variability", "VariabilityStars", "VariabilityGalaxies"]
+__all__ = ["Variability", "VariabilityStars", "VariabilityGalaxies",
+           "reset_agn_lc_cache"]
+
+_agn_lc_cache = {} # a global cache of agn light curve calculations
+
+
+def reset_agn_lc_cache():
```

Comment:
Public function should have a docstring. The variable _agn_lc_cache should probably be all Uppercase according to conventions, and starting with the underscore as done.